color.lisp

```diff
@@ -42,7 +42,7 @@
 ;;    - :fg and :bg take a color as an argument, which can either be a numeric
 ;;      index into the color map or a hexadecimal color in the form of "#fff"
 ;;      or "#ffffff".
-;;    - :reverse and :bright take either t or nil as an argument. T enables
```

Comment:
I really dislike starting sentences with variable names or numbers, e.g. years or dates. It's like starting a dance on the wrong foot or beat. Not that I dance, but as an analogy.